pkg/api/v1/locations_test.go

```diff
@@ -8,169 +8,136 @@ import (
 		},
 		{
 			name:   "get all locations",
 			path:   baseURL,
 			status: http.StatusOK,
+			tenant: tenantID.String(),
+			method: "GET",
 		},
 		{
 			name:   "get inavlid tenant",
-			path:   srv.URL + "/v1/tenant/invalid/locations",
+			path:   baseURL,
 			status: http.StatusBadRequest,
+			tenant: "invalid",
+			method: "GET",
 		},
 		{
 			name:   "get invalid location",
 			path:   baseURL + "/invalid",
 			status: http.StatusNotFound,
+			tenant: tenantID.String(),
+			method: "GET",
 		},
-	}
-
-	for _, tt := range pathTests {
-		t.Run(tt.name, func(t *testing.T) {
-			res, err := srv.Client().Get(tt.path) //nolint:noctx
-			assert.NoError(t, err)
-			assert.Equal(t, tt.status, res.StatusCode)
-			res.Body.Close()
-		})
-	}
-
-	var deleteTests = []struct {
-		name   string
-		path   string
-		status int
-	}{
+		// Delete tests
 		{
 			name:   "delete reef",
 			path:   baseURL + "/reef",
-			status: http.StatusOK,
+			status: http.StatusNoContent,
+			tenant: tenantID.String(),
+			method: "DELETE",
 		},
 		{
 			name:   "delete shell",
 			path:   baseURL + "/shell",
-			status: http.StatusOK,
+			status: http.StatusNoContent,
+			tenant: tenantID.String(),
+			method: "DELETE",
 		},
-	}
-
-	for _, tt := range deleteTests {
-		t.Run(tt.name, func(t *testing.T) {
-			req, err := http.NewRequest("DELETE", tt.path, nil) //nolint:noctx
-			assert.NoError(t, err)
-
-			req.Header.Set("Content-Type", "application/json")
-			res, err := http.DefaultClient.Do(req)
-			assert.NoError(t, err)
-			assert.Equal(t, tt.status, res.StatusCode)
-			res.Body.Close()
-		})
-	}
+	})
 }
 
-func createAnenmoes(t *testing.T, srv *httptest.Server) (*response, func(t *testing.T)) {
+func createAnemones(t *testing.T, srv *httptest.Server) (*response, func(t *testing.T)) {
 	tenantID := uuid.New().String()
-	baseURL := srv.URL + "/v1/tenant/" + tenantID + "/locations"
+	baseURL := srv.URL + "/v1/locations"
 	happyPath := `{"display_name": "anemones"}`
 
 	t.Run("POST anemones", func(t *testing.T) {
-		res, err := srv.Client().Post(baseURL, "application/json", httptools.FakeBody(happyPath)) //nolint:noctx
+		req, err := http.NewRequest("POST", baseURL, httptools.FakeBody(happyPath)) //nolint
 		assert.NoError(t, err)
-		assert.Equal(t, 201, res.StatusCode)
+
+		req.Header.Set("Content-Type", "application/json")
+		req.Header.Set("X-Infratographer-Tenant-ID", tenantID)
+
+		res, err := srv.Client().Do(req)
+		assert.NoError(t, err)
+		assert.Equal(t, http.StatusCreated, res.StatusCode)
 		res.Body.Close()
 	})
 
 	loc := response{}
 
 	t.Run("GET anemones", func(t *testing.T) {
-		res, err := srv.Client().Get(baseURL + "/anemones") //nolint:noctx
+		req, err := http.NewRequest("GET", baseURL, nil) //nolint
+		assert.NoError(t, err)
+
+		req.Header.Set("Content-Type", "application/json")
+		req.Header.Set("X-Infratographer-Tenant-ID", tenantID)
+
+		res, err := srv.Client().Do(req)
 		assert.NoError(t, err)
 		assert.Equal(t, 200, res.StatusCode)
```

Comment:
This is a nit, so feel free to toss it. But should we pick to either use the `int` response codes or the `http.*` constants when checking `StatusCode`?